src/llmcompressor/transformers/finetune/__init__.py

```diff
@@ -2,4 +2,4 @@
 
 from .data import TextGenerationDataset
 from .session_mixin import SessionManagerMixIn
-from .text_generation import apply, compress, eval, oneshot, train
```

Comment:
Can we please keep llmcompressor.transformers.oneshot? It's a very public api and this breaks a lot of users' scripts

Reply:
I think we do want to remove it from transformers as the goal is to kill the transformers folder altogether 
I think we just have to gracefully indicate the old pathway being deprecated (maintain the location under entrypoints but keep llmcompressor.transformers.oneshot a valid pathway, similar to what we did with AutoModel)

Reply:
Can we at least leave a depreciation warning with subsequent redirect?

Reply:
Or do that in a separate PR? 80% of the file changes here are just changing this path

Reply:
I'm fine with either